docsets/Mastodon/docset.json

```diff
@@ -1,13 +1,17 @@
 {
     "name": "Mastodon",
-    "version": "4.2.8/2024-04-26",
+    "version": "4.3.0/2024-10-17",
     "archive": "Mastodon.tgz",
     "author": {
         "name": "Christian Paul",
         "link": "https://github.com/jaller94"
     },
     "aliases": [],
     "specific_versions": [
+        { 
+            "version": "4.3.0/2024-10-17",
+            "archive": "versions/2024-04-26/Mastodon.tgz"
```

Comment:
This file path is wrong. I think it should be `versions/2024-10-17/Mastodon.tgz`

Reply:
```suggestion
            "archive": "versions/2024-10-17/Mastodon.tgz"
```